runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,36 @@
+*nvim-extra-keymaps.txt*    For nvim.                         {Nvim}
+						    Last change:      2015 May
+
+EXTRA KEYMAPS FOR NVIM                                      *nvim-extra-keymaps*
```

Comment:
IDK if this is done in regular Vim, but the nvim docs don't seem to follow this scheme (capital letters, lack of `===========`). See the comment I made on line 17.